evm/parity/Dockerfile

```diff
@@ -10,8 +10,8 @@ RUN apt-get install -y libudev-dev
 
 # install python modules needed for benchmarking script
 RUN pip3 install durationpy jinja2 pandas
-RUN rustup default nightly-2019-01-15 && \
-    rustup target add wasm32-unknown-unknown
+
+RUN rustup default nightly-2019-01-15
```

Comment:
Also not convinced parity compilation requires this, given bench-build-base has rust 1.42.0, which came out March 2020.

Reply:
@jwasinger can you perhaps check removing it and checking the build?

Reply:
I tried removing it and using the version and using the one in bench-build-base, but when trying to build I get this error:

```
error[E0382]: assign to part of moved value: `engine`
   --> ethcore/src/engines/clique/mod.rs:205:4
    |
191 |         let mut engine = Clique {
    |             ---------- move occurs because `engine` has type `engines::clique::Clique`, which does not implement the `Copy` trait
...
202 |         let res = Arc::new(engine);
    |                            ------ value moved here
...
205 |             engine.step_service = Some(StepService::start(Arc::downgrade(&res) as Weak<Engine<_>>));
    |             ^^^^^^^^^^^^^^^^^^^ value partially assigned here after move

```

Reply:
Strange, I also tried compiling it locally but results in a SIGSEGV, sigh.

Anyway this PR is fine to merge as it only reorders stuff. We should not remove setting this specific nightly version.